doc/source/cluster/kubernetes/user-guides/kubectl-plugin.md

```diff
@@ -60,9 +69,250 @@ Forwarding from [::1]:10001 -> 10001
+Cluster is not ready: Cannot determine cluster state
+Cluster is not ready: Cannot determine cluster state
+Cluster is not ready: Cannot determine cluster state
+Cluster is not ready: Cannot determine cluster state
+Cluster is not ready: Cannot determine cluster state
+Cluster is not ready: Cannot determine cluster state
+Cluster is not ready: Cannot determine cluster state
+Waiting for portforwarding...Port Forwarding service rayjob-interactivemode-raycluster-qbkr8-head-svc
+Forwarding from 127.0.0.1:8265 -> 8265
+Forwarding from [::1]:8265 -> 8265
+Handling connection for 8265
+Portforwarding started on http://localhost:8265
+Ray command: [ray job submit --address http://localhost:8265 --working-dir . -- python sample_code.py]
+Running ray submit job command...
+Handling connection for 8265
+Handling connection for 8265
+Handling connection for 8265
+2025-01-06 12:44:41,234	INFO dashboard_sdk.py:338 -- Uploading package gcs://_ray_pkg_3ddba7608d86c45a.zip.
+2025-01-06 12:44:41,238	INFO packaging.py:601 -- Creating a file package for local module '.'.
+Handling connection for 8265
+Handling connection for 8265
+Handling connection for 8265
+Handling connection for 8265
+Handling connection for 8265
+2025-01-06 12:44:41,077	INFO cli.py:39 -- Job submission server address: http://localhost:8265
+2025-01-06 12:44:42,312	SUCC cli.py:63 -- -------------------------------------------------------
+2025-01-06 12:44:42,313	SUCC cli.py:64 -- Job 'raysubmit_fuBdjGnecFggejhR' submitted successfully
+2025-01-06 12:44:42,313	SUCC cli.py:65 -- -------------------------------------------------------
+2025-01-06 12:44:42,313	INFO cli.py:289 -- Next steps
+2025-01-06 12:44:42,313	INFO cli.py:290 -- Query the logs of the job:
+2025-01-06 12:44:42,313	INFO cli.py:292 -- ray job logs raysubmit_fuBdjGnecFggejhR
+2025-01-06 12:44:42,313	INFO cli.py:294 -- Query the status of the job:
+2025-01-06 12:44:42,313	INFO cli.py:296 -- ray job status raysubmit_fuBdjGnecFggejhR
+2025-01-06 12:44:42,313	INFO cli.py:298 -- Request the job to be stopped:
+2025-01-06 12:44:42,313	INFO cli.py:300 -- ray job stop raysubmit_fuBdjGnecFggejhR
+2025-01-06 12:44:42,472	INFO cli.py:307 -- Tailing logs until the job exits (disable with --no-wait):
+2025-01-06 12:44:41,931	INFO job_manager.py:530 -- Runtime env is setting up.
+2025-01-06 12:44:43,542	INFO worker.py:1494 -- Using address 10.12.0.10:6379 set in the environment variable RAY_ADDRESS
+2025-01-06 12:44:43,542	INFO worker.py:1634 -- Connecting to existing Ray cluster at address: 10.12.0.10:6379...
+2025-01-06 12:44:43,551	INFO worker.py:1810 -- Connected to Ray cluster. View the dashboard at 10.12.0.10:8265 
+[0, 1, 4, 9]
+2025-01-06 12:44:47,830	SUCC cli.py:63 -- ------------------------------------------
+2025-01-06 12:44:47,830	SUCC cli.py:64 -- Job 'raysubmit_fuBdjGnecFggejhR' succeeded
+2025-01-06 12:44:47,830	SUCC cli.py:65 -- ------------------------------------------
+```
+
+### Create a Ray Cluster
```

Comment:
Might make sense to show the "Create a Ray cluster" example before "ray job submit" examples, since job submission (without yaml) relies on the cluster creation

Reply:
Done!